tools/devsetup/README.txt

```diff
@@ -36,3 +36,43 @@ macOS support
 -------------
 All required dependencies except virtualenv can be installed via Homebrew,
 virtualenv is installed with pip.
+
+
+Dockerfile
+----------
+
+A Dockerfile is present to showcase the needed installation and how to run the
+app. The base image is based on debian buster, which can also be used for
+development should one wishes to. The admin name for the app and the database,
+as well as the name of the database itself is set to 'pgeusystem'.
+
+The created user, pgeusystem, can be mapped to the userid and groupid of the
+host user that owns the base directory that the app is installed. It is
+recommended that it is not root. That allows for the base directory to be
+mounted in the container and changes to be able to propagate from host to
+container without rebuilding the image.
+
+A typical usecase would be to build the docker image, via:
+	$  docker build -t pgeusystem-image \
+		--build-arg USER_ID=$(id -u) \
+		--build-arg GROUP_ID=$(id -g) \
+		-f ./tools/devsetup/Dockerfile .
+from the base directory.
+
+Then run said image in the background using the host's network and binding the
+base directory in shared mode, via:
+	$ docker run \
+		--mount type=bind,source="$(pwd)",target=/opt/pgeusystem/app,bind-propagation=shared \
+		--name pgeusystem-image \
+		--network host \
+		-d \
+		pgeusystem-image
+
+If the above commands are successfull then one can reach the index page of the
+app in http://localhost:8012
+
+If the user so wishes, can reach the database in the running container via:
```

Comment:
Does that mean you have to login into the container first?
Then it would be useful to add the command how to do that.

Reply:
I suggest above to run the image with the `--network host` parameter. When that is done, and is successful, then it is not necessary to login into the container first.

Reply:
Ah, perfect.